normalisation/normalise_nitro.py

```diff
@@ -13,155 +69,63 @@ def strip_chain_id(string):
-        "0xbB06DCA3AE6887fAbF931640f67cab3e3a16F4dC": "usdt",
-    },
-    "1101": {
-        "0x1E4a5963aBFD975d8c9021ce480b42188849D41d": "usdt",
-        "0xA8CE8aee21bC2A48a5EF670afCc9274C7bbbC035": "usdc",
-        "0x4F9A0e7FD2Bf6067db6994CF12E4495Df938E6e9": "weth",
-    },
-    "5000": {
-        "0x09Bc4E0D864854c6aFB6eB9A9cdF58aC190D0dF9": "usdc",
-        "0x201EBa5CC46D216Ce6DC03F6a759e8E766e956aE": "usdt",
-        "0xdEAddEaDdeadDEadDEADDEAddEADDEAddead1111": "weth",
-    },
-    "8453": {
-        "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913": "usdc",
-        "0x4200000000000000000000000000000000000006": "weth",
-    },
-    "34443": {
-        "0xd988097fb8612cc24eeC14542bC03424c656005f": "usdc",
-        "0xf0F161fDA2712DB8b566946122a5af183995e2eD": "usdt",
-        "0x4200000000000000000000000000000000000006": "weth",
-    },
-    "42161": {
-        "0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9": "usdt",
-        "0xaf88d065e77c8cC2239327C5EDb3A432268e5831": "usdc",
-        "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1": "weth",
-    },
-    "43114": {
-        "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E": "usdc",
-        "0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7": "usdt",
-        "0x8b82A291F83ca07Af22120ABa21632088fC92931": "weth",
-    },
-    "59144": {
-        "0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f": "weth",
-        "0x176211869cA2b568f2A7D4EE941E073a821EE1ff": "usdc",
-        "0xA219439258ca9da29E9Cc4cE5596924745e12B93": "usdt",
-    },
-    "81457": {
-        "0x4300000000000000000000000000000000000004": "weth",
-    },
-    "534352": {
-        "0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4": "usdc",
-        "0xf55BEC9cafDbE8730f096Aa55dad6D22d44099Df": "usdt",
-        "0x5300000000000000000000000000000000000004": "weth",
-    },
-    # "728126428": {
-    #     "TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t": "usdt",
-    #     "TEkxiTehnzSmSe2XqrBj4w32RUN966rdz8": "usdc",
-    #     "TXWkP3jLBqRGojUih1ShzNyDaN5Csnebok": "weth"
-    # },
-    # https://developers.tron.network/docs/account#account-address-format
-    # hex version of the base58 address
-    # can be converted on https://www.btcschools.net/tron/tron_tool_base58check_hex.php
-    "728126428": {
-        "41a614f803b6fd780986a42c78ec9c7f77e6ded13c": "usdt",
-        "413487b63d30b5b2c87fb7ffa8bcfade38eaac1abe": "usdc",
-        "41ec51baf14488ec651270ccc409afda2818af74f2": "weth",
-    },
-}
-
-nitro_token_name_key = {}  # Computed using `nitro_token_address_key`
-
-for chain, tokens in NITRO_TOKEN_ADDRESS_KEY.items():
-    for address, token_name in tokens.items():
-        if chain not in nitro_token_name_key:
-            nitro_token_name_key[chain] = {}
-        nitro_token_name_key[chain][token_name] = address
+    chain_id = strip_chain_id(bytearray.fromhex(byte_string[2:]).decode("utf-8"))
+    if chain_id == "":
+        raise ValueError(f"Invalid chain id bytes (might be expected): [{byte_string}]")
+    if chain_id == "solana":
```

Comment:
Can it really be "solana" string in bytes data?

Reply:
it's not bytes by this point, strip_chain_id returns string